libs/service-portal/settings/personal-information/src/components/Forms/ProfileForm/components/Inputs/PaperMail.tsx

```diff
@@ -0,0 +1,140 @@
+import { useState } from 'react'
+import { gql, useMutation, useQuery } from '@apollo/client'
+import { Query } from '@island.is/api/schema'
+import {
+  Box,
+  Columns,
+  Column,
+  Icon,
+  LoadingDots,
+  Checkbox,
+  Hidden,
+} from '@island.is/island-ui/core'
+import { m } from '@island.is/service-portal/core'
+import { msg } from '../../../../../lib/messages'
+import { useLocale, useNamespaces } from '@island.is/localization'
+import { Controller, useForm } from 'react-hook-form'
+import { FormButton } from '../FormButton'
+import * as styles from './ProfileForms.css'
+
+const GET_PAPER_MAIL = gql`
+  query documentSenders {
+    getPaperMailInfo {
+      wantsPaper
+      nationalId
+    }
+  }
+`
+
+const POST_PAPER_MAIL = gql`
+  mutation PostRequestPaperMutation($input: PostRequestPaperInput!) {
+    postPaperMailInfo(input: $input) {
+      wantsPaper
+      nationalId
+    }
+  }
+`
+interface Props {
+  wantsPaper: boolean
+}
+
+export const PaperMail = () => {
+  useNamespaces('sp.settings')
+  const { formatMessage } = useLocale()
+  const { control, handleSubmit } = useForm<Props>()
+  const [localValue, setLocalValue] = useState<boolean | undefined>(undefined)
+  const [submitError, setSubmitError] = useState<string>()
+
+  const { data, loading } = useQuery<Query>(GET_PAPER_MAIL)
+  const [
+    postPaperMailMutation,
+    { data: mutationData, loading: postLoading },
+  ] = useMutation(POST_PAPER_MAIL)
+
+  const submitFormData = async (data: { wantsPaper: boolean }) => {
+    try {
+      setSubmitError(undefined)
+      await postPaperMailMutation({
+        variables: {
+          input: {
+            wantsPaper: data.wantsPaper,
+          },
+        },
+      })
+    } catch (err) {
+      console.error(`updateOrCreateUserProfile error: ${err}`)
+      setSubmitError(formatMessage(m.somethingWrong))
```

Comment:
No log?

Reply:
Error logged in the service, this is just to display error for the user.